src/Tribe/Views/V2/View.php

```diff
@@ -1533,13 +1535,12 @@ protected function filter_repository_args( array $repository_args, Context $cont
 	 * @return string The View request URI, a value suitable to be used to set the `$_SERVER['REQUEST_URI']` value.
 	 */
 	protected function get_request_uri() {
-		$request_uri = '/' . ltrim(
-				str_replace(
-					home_url(),
-					'',
-					$this->rewrite->get_clean_url( (string) $this->get_url() ) ),
-				'/'
-			);
+		$plain_url   = (string) $this->get_url();
+		$clean_url   = $this->rewrite->get_clean_url( $plain_url );
+		$url_frags   = wp_parse_url( $clean_url );
+		$request_uri = '/' . ( isset( $url_frags['path'] ) ? trim( $url_frags['path'], '/' ) . '/' : '' )
+					   . ( isset( $url_frags['query'] ) ? '?' . $url_frags['query'] : '' )
```

Comment:
:warning: Found precision alignment of 3 spaces.

<sub>hash: c57e5ed91274ef6453330eb50615603d</sub>